lib/ReplacePointerBitcastPass.cpp

```diff
@@ -354,6 +354,11 @@ void ComputeStore(IRBuilder<> &Builder, StoreInst *ST, Value *OrgGEPIdx,
   unsigned DstEleTyBitWidth = SizeInBits(Builder, DstEleTy);
 
   Type *OrigSrcTy = SrcTy;
+  if (dyn_cast<IntToPtrInst>(Src)) {
```

Comment:
Could you add a comment to explain what is happening here, and why it is needed

Reply:
Added a comment with explanation.